fermente_mrp_bom_form_view/models/mrp_bom.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2025 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin DUPONT (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class MrpBom(models.Model):
+    _inherit = "mrp.bom"
+
+    product_name = fields.Char(related="product_tmpl_id.name")
+
+    product_finished = fields.Boolean(
+        string="Finished product",
+        related="product_tmpl_id.sale_ok",
+    )
```

Comment:
```suggestion
    sale_ok = fields.Boolean(
        related="product_tmpl_id.sale_ok",
    )
```

si c'est un related de sale_ok, je serais favorable 
- à utiliser le même terme technique (sale_ok). Plus simple à comprendre à la longue pour les développeurs.
- à ne pas surcharger le string. du coup, le string reprendrait le libellé de sale_ok "Peut-etre vendu".

Qu'en penses-tu ?

Reply:
Ok pour renommer en sale_ok pour faciliter la compréhension champ
Mais ce qui est important pour les users finaux, c'est qu'il y ait leur vocabulaire : les transfos parlent de Produit finis donc ça me paraît important de coller à leur vocabulaire (Produit fini, produit intermédiaire, tous les changements de Nomenclatures en Fiches techniques)

Mais du coup si on renomme le champ en sale_ok, ça fait moins sens de mettre le string en "Finished product" ?